avaje-jex-htmx/src/main/java/io/avaje/jex/htmx/TemplateRender.java

```diff
@@ -0,0 +1,12 @@
+package io.avaje.jex.htmx;
+
+/**
+ * Template render API.
+ */
+public interface TemplateRender {
```

Comment:
should I take out the Template Render in the main module?

Reply:
Er, I'm not sure to be honest.  It could be that we remove the TemplateRender from here ???

I think this is a question is around supporting both "Traditional Templating / server side rendering" and "HTMX specific". 

Umm, thinking ...

Reply:
for me, I'm a fan of the one in the main module because it doesn't support Jstachio as it takes a Map<String, Object>

Reply:
Ok, at this stage we will keep them different. This html TemplateRender is currently designed around use with JStache and ultimately could evolve a little bit (we know the type of the template so could skip a lookup if we wanted to etc).

So yeah I reckon we keep this separate because they are subtly different and there is that chance that this one evolves a bit more.

Reply:
cool, RC time